### DISCUSSION TITLE
4.0.1

### DISCUSSION THREAD

**Discussion by bonanitech** [node-red-contrib-themes/theme-collection#448] [Announcements]
# What’s changed

## 🐛 Bug fixes

* Fix themes directory reading ([#446](https://github.com/node-red-contrib-themes/theme-collection/pull/446)) @bonanitech

## 📖 Documentation

* Fix release-publish workflow badge ([#444](https://github.com/node-red-contrib-themes/theme-collection/pull/444)) @bonanitech

**Full Changelog**: https://github.com/node-red-contrib-themes/theme-collection/compare/4.0.0...4.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/node-red-contrib-themes/theme-collection/releases/tag/4.0.1'>4.0.1</a>.</em>

**Comment by fermacedojr** [comment]
Hey, I'm a big fan of this project. I verified that you have already updated the themes to version 4.0 of Node-RED. However, I missed the pretty scrollbars for dark themes. Do you have plans to implement this?

**Reply by bonanitech** [reply]
@fermacedojr, since [4.0.0](https://github.com/node-red-contrib-themes/theme-collection/releases/tag/4.0.0), the scrollbars are customized differently.

You use a dark theme in Node-RED, and your operating system is on a light theme, right?

**Reply by fermacedojr** [reply]
Hi, @bonanitech . Congratulations for your excellent work and I thank you very much for your personal attention and your quick response. Yes, I'm using it on Windows with Chrome and a light theme. If you don't mind replying, in this case, how should I customize the scrollbars?

**Reply by bonanitech** [reply]
Please allow me some time to consider the options. I'll probably release a new version in the next few days.

**Reply by fermacedojr** [reply]
Looking forward to this new version. Thank you very much for your attention.

**Reply by bonanitech** [reply]
Unfortunately, I haven't found a way to customize only the scrollbar colors consistently across all the browsers. I would have to change their shape to be able to do that. And that's one of the reasons the themed scrollbars were removed for now.

We can revisit this when the `scrollbar-color` CSS property gets supported by all major browsers.

**Reply by fermacedojr** [reply]
@bonanitech , despite this, thanks for your effort and attention on this issue. I will be very happy when you have any news about it. Congrats for all the fine job done on this project. 😀👍

**Reply by bonanitech** [reply]
@fermacedojr, check this out. 😉

https://github.com/node-red-contrib-themes/theme-collection/pull/550
